docs/deployment/server_access_logging.md

```diff
@@ -0,0 +1,40 @@
+---
+id: server_access_logging
+title: S3 Server Access Logging
+hide_title: true
+---
+
+# S3 Server Access Logging
+
+To enable [EMS Reporting](../ems_reporting.md), you need to enable [S3 Server access logging.][awslogging] on all protected buckets.
```

Comment:
```suggestion
To enable [EMS Reporting](../ems_reporting.md), you need to enable [S3 Server access logging][awslogging] on all protected buckets.
```

Remove the period in the link for S3 Server access logging. Also, nit: should "Server" be capitalized?

Reply:
So, I think we should use <screetch>
I was just about say we should use S3 Server Access Logging because that's the title of the page and what amazon uses, but it's not.  It's the title of the page, but in headers it's title cased elsewhere it's just `server access logging`.  I'll fix that.

Reply:
ff9fb8c2f